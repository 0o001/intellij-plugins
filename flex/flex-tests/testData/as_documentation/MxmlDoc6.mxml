<?xml version="1.0" encoding="utf-8"?>
<!--- 
	The class level comment for the component. 
	This tag supports all ASDoc tags, 
	and does not require a CDATA block.

	@see mx.core.UIComponent
-->
<mx:Application xmlns:mx="http://www.adobe.com/2006/mxml" horizontalCenter="">
<mx:Script>
  var z:Mxml<caret>Doc6;
</mx:Script>
</mx:Application>
