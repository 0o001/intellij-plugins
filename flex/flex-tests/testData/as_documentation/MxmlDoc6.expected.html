<div class='definition'><pre>MxmlDoc6</pre></div><div class='content'><p>The class level comment for the component.<p>
This tag supports all ASDoc tags,<p>
and does not require a CDATA block.</div><table class='sections'><tr><td valign='top' class='section'><p>See also:</td><td><a href="psi_element://mx.core.UIComponent">mx.core.UIComponent</a></td></table>